doc/api.rst

```diff
@@ -182,6 +182,7 @@ Computation
    Dataset.groupby_bins
    Dataset.rolling
    Dataset.rolling_exp
+   Dataset.cumulative
```

Comment:
Thoughts on using a verb like `accumulate`?

Reply:
I see it as a bit too numpy-lingo-ish. 

I quite liked `.cumulutive` (I think it was your idea!) — it's the normal way of describing the calc...

Has anyone every said "accumulate the sum"? vs "take a cumulative sum"...